src/components/Register/Register.tsx

```diff
@@ -1,286 +1,94 @@
+                                width="28rem"
+                            />
+                            <TextInput
+                                placeholder="Passwort"
+                                onChange={(value) => registerStore.setPassword(value.target.value)}
+                                className="text-input"
+                                height="2.69rem"
+                                width="28rem"
+                            />
+                            <TextInput
+                                placeholder="Passwort bestätigen"
+                                onChange={(value) => registerStore.setRetypedPassword(value.target.value)}
+                                className="text-input"
+                                height="2.69rem"
+                                width="28rem"
+                            />
+                            <div className="account-type flex-row-box">
+                                <Button
+                                    name="Lehrer"
+                                    onClick={() => registerStore.setAccountType('teacher')}
+                                    width="13.5rem"
+                                    height="2.69rem"
+                                    fontSize="22px"
+                                    styleType="unset"
+                                />
+                                <Button
+                                    name="Schüler"
+                                    onClick={() => registerStore.setAccountType('pupil')}
+                                    width="13.5rem"
+                                    height="2.69rem"
+                                    fontSize="22px"
+                                    styleType="unset"
+                                />
+                            </div>
                         </div>
-                    </form>
+                        <div className="actions flex-row-box">
+                            <input
+                                type="checkbox"
+                                onClick={() => registerStore.setIsAgbAccepted(registerStore.isAgbAccepted)}
+                            />
+                            <span>AGB Text. Ich habe die Datenschutzbestimmungen gelesen und akzeptiert.</span>
+                        </div>
+                        <div className="submit-region flex-row-box">
+                            {registerStore.accountType === 'teacher' ? (
+                                <TextInput
+                                    placeholder="Teacher Token"
+                                    onChange={(value) => registerStore.setTeacherToken(value.target.value)}
+                                    className="text-input teacher-token"
+                                    height="2.69rem"
+                                    width="15rem"
+                                />
+                            ) : null}
+                            <Button
```

Comment:
no action when clicking on the button, verify all values and redirect to dashboard